include/fmt/compile.h

```diff
@@ -15,6 +15,12 @@
 #include "format.h"
 
 FMT_BEGIN_NAMESPACE
+
+FMT_BEGIN_EXPORT
+// A compile-time string which is compiled into fast formatting code.
+class compiled_string {};
+FMT_END_EXPORT
```

Comment:
Since it's a single definition let's use `FMT_EXPORT` instead of `FMT_(BEGIN|END)_EXPORT` here.

Reply:
Done